controller.py

```diff
@@ -0,0 +1,30 @@
+import processor
+import view
+import variable
+import sys, os
+clear = lambda: os.system('cls')
+
+
+def start():
+    clear ()
```

Comment:
Вы это используете всего 1 раз. Можно было прямо тут и прописать
```suggestion
    os.system('cls')
```